conda-forge.yml

```diff
@@ -1,6 +1,14 @@
+build_platform:
+  linux_aarch64: linux_aarch64
+  linux_ppc64le: linux_ppc64le
```

Comment:
Note that `linux_ppc64le` is untested besides being importable. It's offered here for the sake of feature parity with h5py.